trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -215,16 +220,25 @@ The `OWS` rule defines an optional whitespace. It is used where zero or more whi
 
 The `tracestate` field value is a `list` as defined below. The `list` is a series of `list-members` separated by commas `,`, and a `list-member` is a key/value pair separated by an equals sign `=`. Spaces and horizontal tabs surrounding `list-member`s are ignored. There can be a maximum of 32 `list-member`s in a `list`.
 
+Empty and whitespace-only list elements are allowed as a list value. But the use
```

Comment:
I recommend using consistent terminology: list members or list elements, not both.